i18n/locales/ja.json

```diff
@@ -14,6 +14,37 @@
         "feedback": "ご意見・ご質問",
         "clickHere": "こちらをクリック"
     },
+    "hamburgerMenu": {
+        "about": "About Find a Doc, Japan",
+        "home": "Home",
+        "contact": "Contact",
+        "submit": "Submit",
+        "privacy": "Privacy",
+        "terms": "Terms",
+        "github": "Github",
+        "netlify": "Powered by Netlify"
+    },
+    "privacyPage": {
```

Comment:
these changes should realistically be a separate PR

Reply:
for the sake of time, I won't block on it, but realistically should try to make more atomic PRs

Reply:
@ermish Agreed! These were only included, because we noticed they were missing while adding the new strings. I think we might want to implement an i18n plugin like we had in 1.0 to make the locale files easier to maintain.